hexa/pipelines/schema/mutations.py

```diff
@@ -43,11 +53,33 @@ def resolve_create_pipeline(_, info, **kwargs):
         }
 
     try:
-        pipeline = Pipeline.objects.create(
-            code=input["code"],
-            name=input.get("name"),
-            workspace=workspace,
-        )
+        notebook = input.get("notebook")
+        if notebook:
+            get_bucket_object(workspace.bucket_name, notebook)
+            pipeline = create_pipeline_object(
+                code=input["code"],
+                name=input.get("name"),
+                workspace=workspace,
+                type=PipelineType.NOTEBOOK,
+                notebook=notebook,
+            )
+            # automatically create a new version
+            pipeline.upload_new_version(
+                user=request.user,
+                name=input.get("name"),
+                zipfile=base64.b64decode("".encode("ascii")),
```

Comment:
I don't see why you have to fill this since it's a notebook type

Reply:
The zipFile field was required in the PipelineVersion model, but yeah it's better to modify the binaryField and allow  it to be null. WDYT?